### ISSUE TITLE
Configuration Management problems with the use of rpi-gpio "hack"; just use lgpio native instead?

### ISSUE THREAD

**Issue by frank-pet** [adafruit/Adafruit_Blinka#910]
### Board Name

Raspberry PI 5

### Steps

A pip update of Adafruit_blinka from a version earlier than 0.8.28(?) to any later version does not work.  This is due to the fact that the rpi-gpio replacement for RPi.GPIO is only installed from the ./adafruit/Raspberry-Pi-Installer-Scripts/master/raspi-blinka.py installer.  It took quite a while to track this down.

### Description

I'd like to propose an alternate solution.  Since the /dev/gpiochip* files and lgpio python package are now part of the Debian kernel 1:6.1.63-rpi7-rpi-v8, simply implement the ./microcontroller/bcm283x/pin.py and ./microcontroller/bcm283x/pwmio/PWMOut.py using lgpio directly, and remove the RPi.GPIO code (3 lines) from the ./microcontroller/bcm2711/pin.py package.   An updated version is available in pull request 911 and has been tested successfully on a PI5 on this kernel and a PI3B as well, using the pin package directly and also using digitalio.DigitalInOut with no changes to the digitalio package.   Also successfully tested the blinka pwmio.PWMOut package. on both platforms.

### Additional information

Code changes in pull request 911

**Comment by frank-pet** [comment]
Hmmm, didn't expect to see the code formatted that way; I can just send you the file if you wish.

**Comment by makermelissa** [comment]
Yeah, this is something I've considered. It would certainly solve one of the installation issues with the RPi.GPIO being installed and causing a conflict. The bcm2711 folder is also used by the Pi 4, which I'm not sure if lgpio would run on, so it may just need a bcm2712 version for the Pi 5 if it doesn't work on the Pi 4.

**Comment by frank-pet** [comment]
I haven’t tested these change on a PI 4, but I don’t see why it wouldn’t work.  The kernel  on my bullseye install shows the /dev/gpio chip devices, so it should run just like it does on the PI3. The conflict you mentioned is a real mess to deal with.   Just had to deal with it again updating a Pi zero w.

**Comment by makermelissa** [comment]
Fixed by #911
